testsuite/gnat2goto/tests/floating_point_literals/floating_point_literals.adb

```diff
@@ -0,0 +1,7 @@
+procedure Floating_Point_Literals is
+  X : Float := 1.0;
```

Comment:
My IEEE knowledge is weak... But is there any value in also having tests with values that are fractional ?

Reply:
I have written some unit tests for the ureal_to_binary conversion here: https://github.com/diffblue/gnat2goto/pull/27/files

This test is more or less just to check that this code path is actually used for float literals.